_includes/header.html

```diff
@@ -2,26 +2,33 @@
 
   <div class="wrapper">
 
-    <a class="site-title" href="{{ site.baseurl }}/index">{{ site.title }}</a>
+    <nav class="navbar navbar-expand-lg navbar-light bg-light">
```

Comment:
Ah, I was thinking you would react-bootstrap, given the nature of this site :-) 
So, are these raw bootstrap components?

Reply:
Oh I can totally do that. I didn't know if you wanted a basic Navbar or something React specific.